recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
         - libdap4 3.18.*  # [not win]
         - kealib
         - libtiff 4.0.*
-        - libpng >=1.6.21,<1.7
+        - libpng >=1.6.23,<1.7
```

Comment:
In my test `>=1.6.23,<1.7` is OK on OS X while `>=1.6.21,<1.7` is known to fail.